app/models/event/application.rb

```diff
@@ -0,0 +1,82 @@
+# frozen_string_literal: true
+
+# == Schema Information
+#
+# Table name: event_applications
+#
+#  id                  :bigint           not null, primary key
+#  aasm_state          :string
+#  address_city        :string
+#  address_country     :string
+#  address_line1       :string
+#  address_line2       :string
+#  address_postal_code :string
+#  address_state       :string
+#  airtable_status     :string
+#  description         :text
+#  name                :string
+#  notes               :text
+#  political           :boolean
+#  referral_code       :string
+#  referrer            :string
+#  created_at          :datetime         not null
+#  updated_at          :datetime         not null
+#  airtable_record_id  :string
+#  event_id            :bigint
+#  user_id             :bigint           not null
```

Comment:
@polypixeldev and i talked about making this a one-to-many, but that's low priority and can be migrated in the future.

we'll keep it one to one for now